deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -90,6 +90,9 @@ master:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name:
+  
+  # specify how many old ReplicaSets for the Deployment to retain.
+  revisionHistoryLimit: 3
```

Comment:
I think you should add the field description in the file located under `docs/deployment/helm.md`

Reply:
oh I didn't add the docs lol .